interface/main/left_nav.php

```diff
@@ -1053,7 +1053,22 @@ function selpopup(selobj) {
 <form method='post' name='find_patient' target='RTop'
  action='<?php echo $rootdir ?>/main/finder/patient_select.php'>
 
-<?php if ( ( $GLOBALS['concurrent_layout'] == 2) || ($GLOBALS['concurrent_layout'] == 3) ) { ?>
+<?php if ( ( $GLOBALS['concurrent_layout'] == 2) || ($GLOBALS['concurrent_layout'] == 3) ) {
+  // mdsupport - menu showhide arrows
+  if ($_SESSION['language_direction'] == 'ltr') {
```

Comment:
For testing we had just changed lang_is-rtl for 'en' in lang_languages to 1 and it seemed to behave as expected.